### ISSUE TITLE
Change name of methods/ folder

### ISSUE THREAD

**Issue by kelliemac** [FredHutch/VISCtemplates#229]
The methods/ folder that houses lab-methods.Rmd, statistical-methods.Rmd, etc. often ends up housing child Rmd documents for other report sections that aren't "methods". I propose changing this folder to be called "Rmd/" (to include all child Rmd documents) and also adding a "R/" folder for source code in the form of .R files.

@asatofh and @mayerbry what do you think about this? I'm not tied to the "Rmd/" name so I'm open to other suggestions if you have them.

**Comment by mayerbry** [comment]
I agree about methods. I'm slightly concerned that multiple `R/` folders at the top-level and sub-directories could cause issues, but I don't have a better idea. Maybe `child-docs` instead of Rmd to be more general (quarto, images), but Rmd is probably fine.

**Comment by kelliemac** [comment]
thanks @mayerbry! I like `child-docs`. an alternative for `R/` could be `src/`.
